### PR TITLE
Agreement content changes

### PR DESCRIPTION
**Pull request by RonanFitzgerald**
I've made these changes based on stakeholder review, it should now match the prototype but I did have to do a lot of tinkering without a preview. It'd be good to review this with a dev.

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/features/offer/views/declaration.njk [hunk 1]
```diff
@@ -45,35 +43,34 @@
...
             countyCourtBusinessCentreUri: 'https://courttribunalfinder.service.gov.uk/courts/county-court-business-centre-ccbc'
           }) | safe
         }}
       </p>
 
-      <p>{{ t('The original claim and any counterclaims will be kept on hold in case this agreement is broken. They will be closed three months after the last payment date (if applicable) if we don’t hear from either of you.') }}</p>
-
       <h2 class="heading-medium">{{ t('Sign the agreement') }}</h2>
 
+      <p>{{t('Make sure this agreement includes everything you've agreed with Mary Richards before signing. You won't be able to change this later.
```
Comment (timja-kainos) [inline src/main/features/offer/views/declaration.njk]: could you add spacing after the {{ and before the t and the same thing at the end please
```
{{ t('
``` 
and
```
') }}
```

File: src/main/features/offer/views/declaration.njk [hunk 2]
```diff
@@ -45,35 +43,34 @@
...
         <li>{{ t('request a judgment for any money still owed from this agreement') }}</li>
-        <li>{{ t('restart the original claim or counterclaim') }}</li>
+        <li>{{ t('restart the original claim or counterclaim for the full amount of the claim') }}</li>
+        <li>{{ t('ask the court to enforce any of the steps to be taken')}}
       </ul>
 
       <p>
         {{
-          t('<a href="{{ n244formUri }}">Complete form N244</a> and send it to the <a href="{{ countyCourtBusinessCentreUri }}">County Court Business Centre</a>. You need to say on the form which of the 2 options you’re applying for.', {
+          t(' To apply, <a href="{{ n244formUri }}">complete form N244</a> and send it to the <a href="{{ countyCourtBusinessCentreUri }}">County Court Business Centre</a>. You need to say on the form which of the 3 options you’re applying for.', {
```
Comment (timja-kainos) [inline src/main/features/offer/views/declaration.njk]: can you remove the extra space at the start please?

Comment (RonanFitzgerald) [reply src/main/features/offer/views/declaration.njk]: I've done that now - also fixed the code for the defendant name. Would be good to see this before we push live, maybe tomorrow ideally, I can't preview while editing in live due to memory issues which makes it a bit tricky, especially with a page like this which was quite different to the prototype.

**Comment by geek-soft** [comment]
Blocked by https://github.com/hmcts/cmc-citizen-frontend/pull/401 due to potential merge conflicts.

**Comment by timja-kainos** [comment]
@RonanFitzgerald This says WIP on it is there anything left to do on it?

**Comment by RonanFitzgerald** [comment]
@timja-kainos this is done from my pov - it'd be good to see a screengrab of it before we finally sign it off tho as i haven't been able to preview how it looks in prod.

**Comment by timja-kainos** [comment]
Has been reviewed by PO

**Pull request merged**